backend/scripts/migrate_data.py

```
@@ -0,0 +1,78 @@
+                                book_id=book.id,
+                                index=chapter_data["index"],
+                                title=chapter_data["title"],
+                                audio_file=chapter_data["audio_file"]
+                            )
+                            db.add(chapter)
+            print("Books migrated successfully.")
+        
+        db.commit()
+        print("Migration completed successfully.")
+        
+    except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)